pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -515,19 +525,24 @@ def preprocess(self, execution_config: ExecutionConfig = DefaultExecutionConfig)
         * Currently does not intrinsically support parameter broadcasting
 
         """
-        config = self._setup_execution_config(execution_config)
+        exec_config = self._setup_execution_config(execution_config)
```

Comment:
Is this variable name update necessary for Catalyst? I'm just curious, I think this is a nice update.

Reply:
no, this is not! I was looking for `config` variables and found this a bit confusing with the `config` attribute that Catalyst uses.